include/text.php

```diff
@@ -1829,8 +1830,7 @@ function file_tag_update_pconfig($uid, $file_old, $file_new, $type = 'file') {
 
 	if (!intval($uid)) {
 		return false;
-	}
-	if ($file_old == $file_new) {
+	} elseif ($file_old == $file_new) {
```

Comment:
I don't think the `elseif` is warranted here. Multiple `if` are fine in my view.

Reply:
Maybe we have different code-writing practices learned ... ;-)

````
function foo ($barEnabled)
{
    if (!is_bool($barEnabled)) {
        throw new InvalidArgumentException(sprintf('barEnabled[]=%s is not expected.', gettype($barEnabled)));
    }

    // [Actual code here]
}
````
... is what I commonly do under PHP.

Reply:
It'd be great, but we don't have a great type awareness in the code generally. And Exception aren't handled at all, which invariably completely crashes the site.

Reply:
That is then a good reason to fix the exception. :-)

Reply:
This is exactly what #4889 is about, I introduced an Exception in #4951 and now we have to fix it for the next release. 😈